src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+
+.nav {
+  display: inline-block;
+  position: absolute;
+  right: 50px;
+}
+
+.logoLink {
+  display: inline-block;
+  margin: 10px 0 10px 50px;
+}
```

Comment:
Better to set horizontal paddings for `header` in one place instead of adding margins for the logo and nav separately